frontend/src/metabase/visualizations/lib/RowRenderer.js

```diff
@@ -16,7 +16,6 @@ import { getFriendlyName } from "./utils";
 import { checkXAxisLabelOverlap } from "./LineAreaBarPostRender";
 
 const ROW_GAP = 5;
-const ROW_MAX_HEIGHT = 30;
```

Comment:
@paulrosenzweig Was there any particular reason to limit the height of the rows?

Reply:
Pretty sure this is ancient history, predating Paul. I have a faint memory of this, and I believe it was possibly to prevent situations e.g. where you only have one or two bars on a large monitor, and the bars become monstrously large. To be clear, I think the "before" image in this PR looks bad, but we should probably check out some scenarios where the bars could become way too big.

Reply:
@mazameli An alternative is to still limit the vertical size of each row (to avoid that monstrously large visual), but at the same  time, align the bars _vertically_ (i.e. equal space above and below those bars).

Reply:
Yeah, or perhaps additionally even out the space between the bars. (Equal space above, below, and in between.)